scripts/mkcloud

```diff
@@ -594,9 +597,52 @@ function scp_install_chef_suse_override()
     fi
 }
 
+function crowbar_dev_init()
+{
+    mkdir -p $host_crowbar_dev_dir
+    pushd $host_crowbar_dev_dir >/dev/null
+    git clone git@github.com:crowbar/crowbar.git
+    pushd $host_crowbar_dev_dir/crowbar >/dev/null
+    ensure_packages_installed ruby2.1-rubygem-bundler ruby2.1-rubygem-rake
+    bundle install
+    rake crowbar:init
+    popd >/dev/null && popd >/dev/null
+}
+
+function crowbar_dev_update()
+{
+    [[ ! -d $host_crowbar_dev_dir ]] && complain 101 "Please run crowbar_dev_init step before."
+    pushd $host_crowbar_dev_dir/crowbar >/dev/null
+    rake crowbar:update
+    popd >/dev/null
+}
+
+function crowbar_dev_guard()
+{
+    pushd $host_crowbar_dev_dir/crowbar >/dev/null
+    if [[ "$wantedcmds" == "crowbar_dev_guard" ]]; then
+        GUARD_SYNC_HOST=$adminip GUARD_TREE_TARGET=$admin_crowbar_dev_dir GUARD_SCRIPT_TARGET=$admin_crowbar_dev_dir/bin bundle exec guard --no-notify <&3
+    else
+        GUARD_SYNC_HOST=$adminip GUARD_TREE_TARGET=$admin_crowbar_dev_dir GUARD_SCRIPT_TARGET=$admin_crowbar_dev_dir/bin bundle exec guard --no-notify
+    fi
+}
+
+function crowbar_dev_install()
+{
+    onadmin crowbar_dev_install
```

Comment:
We dont really need these wrapper functions anymore, because you can now call this step as onadmin+crowbar_dev_install